react_config

```diff
@@ -1,47 +1,50 @@
-#!/bin/bash
-#
-INSTANCE="ubuntu"
-UNAMEX="ubuntu"
-WEBSITENAME="nsfcareer-react-website"
-
-cd /home/$UNAMEX
-echo $UNAMEX
-#sudo apt-get update && sudo apt-get upgrade -y
-sudo apt-get install nginx -y
-sudo systemctl enable nginx
-# sudo apt-get update -y
-sudo apt-get install build-essential libssl-dev -y
-curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
-sudo apt-get install -y nodejs
-sudo apt-get install -y  npm
-sudo npm i -g npx
-cd /home/$UNAMEX
-git clone https://github.com/PSUCompBio/nginx-digitalbrain.git
-sudo cp nginx-digitalbrain/default /etc/nginx/sites-available/
-#
-# at this step you need to update A record on google domains
-#
-sudo systemctl reload nginx
-
-git clone https://github.com/PSUCompBio/nsfcareer-react-website.git
-cd /home/$UNAMEX/$WEBSITENAME
-sudo npm install
-sudo npm install -g concurrently
-sudo npm install -g nodemon
-sudo npm run init
-# sudo npm run build
-sudo npm install -g serve
-sudo npm install pm2 -g
-sudo npm i -g babel-cli
-export PM2_HOME=/home/$UNAMEX/.pm2
-sudo pm2 start -u $UNAMEX --interpreter babel-node  client/node_modules/react-scripts/scripts/start.js  --name="app"
-# sudo pm2 serve build -u $UNAMEX --name="app"
-cd /home/$UNAMEX
-cp /home/$UNAMEX/$WEBSITENAME/Website-Directions.readme .
-sudo chown -R $UNAMEX *
-#sudo add-apt-repository ppa:certbot/certbot -y
-#sudo apt-get update -y
-#sudo apt-get install python-certbot-nginx -y
-#sudo ufw status
-#sudo ufw allow 'Nginx Full'
-#sudo certbot --nginx -d digitalbraininjury.com -d www.digitalbraininjury.com --non-interactive --agree-tos -m reuben.kraft@gmail.com
+#!/bin/bash
```

Comment:
I am not sure why this is showing that everything was removed and then added? Were there any changes to this file?

Reply:
Only pm2 commands were updated and cd command was added.